### PR TITLE
Toggle for dark / light mode

### PR DESCRIPTION
**Pull request by Shivareddy-Aluri**
Implemented a toggle for dark and light mode. used localStorage to achieve it!
just added a button as toggle now, will search for an dark mode svg and add it once the code review is done!

closes: https://github.com/MelSumner/a11y-automation/issues/409

https://user-images.githubusercontent.com/59643926/206921561-1c2667d0-4e6c-4740-9c29-a3668712801a.mov

update  added icon too:

Screenshot 2022-12-11 at 1 58 47 PM

Screenshot 2022-12-11 at 1 59 22 PM

### CODE REVIEW

**Comment by Shivareddy-Aluri** [comment]
@MelSumner can you please review this :)

File: app/components/aa-navbar.hbs [hunk 1]
```diff
@@ -13,33 +13,50 @@
...
+              {{else}}
+                <img
+                  src="/assets/images/dark-theme.svg"
+                  alt=""
+                  role="presentation"
+                  height="20"
+                  width="20"
+                />
+              {{/if}}
+            </button>
```
Comment (MelSumner) [inline app/components/aa-navbar.hbs:59]:
```suggestion
          <button
              type="button"
              aria-label="toggle dark/light mode"
              {{onclick this.onChangeTheme}}
              class="bg-white px-3 py-2 rounded-md ml-80">
              {{#if this.isDarkTheme}}
                <img
                  src="/assets/images/light-theme.svg"
                  alt=""
                  role="presentation"
                  height="20"
                  width="20"
                />
              {{else}}
                <img
                  src="/assets/images/dark-theme.svg"
                  alt=""
                  role="presentation"
                  height="20"
                  width="20"
                />
              {{/if}}
            </button>
```

**Comment by MelSumner** [comment]
@Shivareddy-Aluri in the preview I see the toggle button changing icons, but I don't actually see the theme itself changing. I think we need to troubleshoot further.

**Comment by Shivareddy-Aluri** [comment]
> @Shivareddy-Aluri in the preview I see the toggle button changing icons, but I don't actually see the theme itself changing. I think we need to troubleshoot further.
no theme too changes.... what preview you are reffering to?  in home page its dark for the upper portion is that what miss-leading you?

File: app/components/aa-navbar.hbs (hunk 1 above)
Comment (Shivareddy-Aluri) [reply app/components/aa-navbar.hbs]: done!

**Comment by Shivareddy-Aluri** [comment]
@MelSumner i didn't get you concern here, please let me know i will fix it !

**Comment by MelSumner** [comment]
Sure, you can check the preview out here: https://deploy-preview-418--a11y-automation-tracker.netlify.app/ 

If you toggle the button, you'll see that the button changes but not the theme. 

Now, this is probably because this is a static site, which means you'd need to trigger a refresh with the button.

I apologize that I haven't had more time to give more explicit guidance, but I should have some extra time over the holidays. Thanks for helping out! 👍

**Comment by Shivareddy-Aluri** [comment]
@MelSumner fixed it!

**Pull request merged**